recipes/stir/build.sh

```diff
@@ -0,0 +1,8 @@
+mkdir build && cd build
+cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
+      -D PYTHON_DEST=$SP_DIR \
```

Comment:
```suggestion
      -D PYTHON_DEST:PATH=$SP_DIR \
```

Reply:
Thanks! What is the rationale behind this change?

Reply:
It's generally better to indicate the type I believe (clearer anyway). It also means it becomes a cached variable. That possibly doesn't matter for conda as presumably the build gets thrown away, but good practice...